src/index.css

```diff
@@ -1,32 +1,49 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;800&family=VT323&display=swap');
 
 :root {
-  --primary: #12bca2;
-  --secondary: #6d15df;
+  --primary: #1f1f1f;
+  --secondary: #3D7EAA;
 }
-h1,h2,h3 {
+
+/* --secondary: linear-gradient(to bottom right, #f0c27b, #4b1248) */
+
+
+h1,
+h2,
+h3 {
```

Comment:
Unless you're resetting the style – which if you do, you should probably do in a separate `reset.css` file or something –, avoid selecting elements directly, prefer selecting classes.